### ISSUE TITLE
EPROTO error while connecting

### ISSUE THREAD

**Issue by Zefau** [koalazak/dorita980#95]
Hi there!

I have implemented an adapter using this library for an IoT platform (https://github.com/Zefau/ioBroker.roomba).

My users recently report that they are not able to connect to the Roomba anymore having the following error message:
```
{"errno":"EPROTO","code":"EPROTO","syscall":"write"}
```

They're config: Roomba 960 with FW 2.4.6-3 on Node.js v10.16.3
My config works though: Roomba 980 with FW v2.4.6-3 on Node.js v10.16.1

We have tried different stuff like https://forum.iobroker.net/post/291518 and https://forum.iobroker.net/post/291586 but it does not seem to work.

Do you have any ideas?
Thanks in advanced

**Comment by koalazak** [comment]
can you run `node -p process.versions | grep openssl` in a box with the error? also in a box that it is working fine?

thanks

**Comment by Zefau** [comment]
Both the one working and the one that fails with the error have the output
```
openssl: '1.1.1c',
```

**Comment by koalazak** [comment]
mmmHmmM
can they try with this connection options?
```
var options = {
    port: 8883,
    clientId: user,
    rejectUnauthorized: false,
    protocolId: 'MQTT',
    protocolVersion: 4,
    ciphers: 'AES128-SHA256',
    clean: false,
    username: user,
    password: password
  };
```
(note the ciphers value)

**Comment by koalazak** [comment]
all reports are for  `960` robot? or do you have other models with the problem?

**Comment by Zefau** [comment]
Works fine for my 980 as well. Users reporting the issue having a 960. Have had no other users with other Roomba's reporting so far.

**Comment by Zefau** [comment]
The suggested settings works! Great! Thanks. Can you implement that?

**Comment by koalazak** [comment]
nice! implemented in dorita980@3.1.4
thanks for reporting
